ground/src/test/java/com/google/android/ground/ui/map/gms/mog/MogImageMetadataTest.kt

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.ground.ui.map.gms.mog
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Assert.assertThrows
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class MogImageMetadataTest {
+
+  // TileCountX = 5, TileCountY = 5
+  private val testMogImageMetadata =
+    MogImageMetadata(
+      originTile = TileCoordinates(10, 10, 10),
+      tileWidth = 100,
```

Comment:
More realistic values would be:
tileWidth/Height: 256x256
imageWidth/Height: multiple of tile width/height, eg 1024x1024